scripts/labs.txt

```diff
@@ -2,7 +2,6 @@ c01-e01
 c01-e03
 c01-e04
 c01-git01
-c01-git02
```

Comment:
if we won't have the c01-git02 exercise as mandatory, should we change the exercise numbers to not have a gap in the exercises numbers or we are comfortable, but having this kind of gaps in multiple classes(can be the case in the future...)

Reply:
I don't see a problem having this gap in the numbers, otherwise it'd be necessary to always rename all answers from students every time an exercise name changes